posthog/api/capture.py

```diff
@@ -403,6 +403,14 @@ def get_event(request):
         else:
             events = [data]
 
+        if not all(data):  # Check that all items are truthy (not null, not empty dict)
```

Comment:
> The request gets a 400 eventually, but triggers several exceptions in its way there.

So just to double check, this isn't a behavior change? I could imagine us filtering out bad events and keeping the good ones.

Reply:
Right now, such a payload will get a 400 with:
```
{
    "type": "validation_error",
    "code": "invalid_payload",
    "detail": "Invalid payload: All events must be dictionaries not 'NoneType'!",
    "attr": null
}
```

this check is in [is_unprocessed_snapshot_event](https://github.com/PostHog/posthog/blob/9576fab1e4720a643ba5e4feec9ba83975c9c6d3/posthog/session_recordings/session_recording_helpers.py#L232). With this PR, the response changes to:
```
{
    "type": "validation_error",
    "code": "invalid_payload",
    "detail": "Invalid payload: some events are null",
    "attr": null
}
```